.github/workflows/transformDataToViews.yml

```diff
@@ -12,6 +12,8 @@ on:
 jobs:
   transformAndPush:
     runs-on: windows-latest
+    env:
+      COMMIT_FORMAT: "Generated from %h%n%nCommit message:%n%s%n%b"
```

Comment:
Maybe a comment here to explain "this is a git '--pretty=format' string: (%h is SHA, %n is newline, %s is commit message subject, %b is commit message body)"